File: e010.py
## 2 million 
## 2000000

from math import sqrt

arr = [2, 3]

for x in range(5, 2000000, 2):
    
    counter = 0
    
    ##for y in range(2, x, 1):
    for y in range(2, int(sqrt(x)) + 1):
        if x % y == 0:
            counter += 1
            break
    
    if counter == 0:
        arr.append(x)

print(sum(arr))

## x 142913828925
## 142913828922

File: e010.c
#include <stdio.h>
#include <math.h>

int main()
{
    long somme = 5;
    int a = 5;
    int z = 2000000;
    int i, count;
    while (a < z)
    {
        count = 0;
        i = 2;
        while (i < floor(sqrt(a)) + 1)
        {
            if (a % i == 0)
            {
                count++;
                break;
            }
            i++;
        }
        if (count == 0)
        {
            somme += a;
        }
        a += 2;
    }
    
    printf("%ld", somme);

    return (0);
}

